tests/integration-tests/tests/update/test_update.py

```diff
@@ -412,6 +413,60 @@ def test_update_awsbatch(region, pcluster_config_reader, clusters_factory, test_
     _verify_initialization(region, cluster, updated_config)
 
 
+@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux2", "slurm")
```

Comment:
not needed for new tests, we are going to remove this mark for old tests as well

Reply:
removed